e2e/test/scenarios/models/models-list-view.cy.spec.js

```diff
@@ -206,13 +206,19 @@ describe("scenarios > models list view", () => {
       cy.findByTestId("list-view-preview").as("listPreview");
       cy.findByTestId("sidebar-right").as("sidebarRight");
 
+      cy.log("Check that used column is not present in unused columns list.");
+      cy.get("@sidebarRight").within(() => {
+        cy.findByText("PRODUCT_ID").should("not.exist");
+      });
```

Comment:
Nit: no need for `.within()` if there's only 1 assertion in it

```suggestion
      cy.get("@sidebarRight").findByText("PRODUCT_ID").should("not.exist");
```